enrollment-server-onboarding/src/main/resources/application.properties

```diff
@@ -149,6 +149,8 @@ enrollment-server-onboarding.presence-check.iproov.ensureUserIdValueEnabled=${IP
 enrollment-server-onboarding.presence-check.iproov.riskProfile=${IPROOV_RISK_PROFILE:}
 enrollment-server-onboarding.presence-check.iproov.serviceBaseUrl=${IPROOV_SERVICE_BASE_URL}
 enrollment-server-onboarding.presence-check.iproov.serviceUserAgent=Wultra/OnboardingServer
+enrollment-server-onboarding.presence-check.iproov.oAuthUsername=${IPROOV_OAUTH_USERNAME:}
```

Comment:
I would prefer `oauthClientUsername` and `oauthClientPassword` to make sure it is clear this is used for client authentication during OAuth.